packages/hardhat-core/sample-projects/advanced-ts/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  env: {
+    browser: false,
+    es2021: true,
+    mocha: true,
+    node: true,
+  },
+  plugins: ["@typescript-eslint"],
+  extends: [
+    "standard",
+    "plugin:prettier/recommended",
+    "plugin:node/recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  overrides: [
+    {
+      files: ["scripts/**"],
+      rules: { "no-process-exit": "off" },
+    },
+    {
+      files: ["hardhat.config.ts"],
+      globals: { hre: true },
+    },
```

Comment:
This is not needed

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1770/commits/1cc3ba38563c9ea6a13788065780801fc52bacbf